frontend/awx/resources/templates/WorkflowVisualizer/components/JobTemplateDetails.tsx

```diff
@@ -0,0 +1,395 @@
+import { useTranslation } from 'react-i18next';
+import {
+  Label,
+  LabelGroup,
+  TextList,
+  TextListItem,
+  TextListItemVariants,
+  TextListVariants,
+} from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
+import { PageDetail, useGetPageUrl, TextCell } from '../../../../../../framework';
+import { useGet } from '../../../../../common/crud/useGet';
+import { JobTemplate } from '../../../../interfaces/JobTemplate';
+import { AwxRoute } from '../../../../main/AwxRoutes';
+import { jsonToYaml } from '../../../../../../framework/utils/codeEditorUtils';
+import { AwxItemsResponse } from '../../../../common/AwxItemsResponse';
+import { useVerbosityString } from '../../../../common/useVerbosityString';
+import { CredentialLabel } from '../../../../common/CredentialLabel';
+import { Credential } from '../../../../interfaces/Credential';
+import { InstanceGroup } from '../../../../interfaces/InstanceGroup';
+import { Label as ILabel } from '../../../../interfaces/Label';
+import { parseStringToTagArray } from '../../JobTemplateFormHelpers';
+import { GraphNodeData, PromptFormValues } from '../types';
+import { useGetTimeoutString } from '../hooks';
+import { NodeCodeEditorDetail } from './NodeCodeEditorDetail';
+import { NodeTagDetail } from './NodeTagDetail';
+import { PromptDetail } from './PromptDetail';
+
+function useAggregateJobTemplateDetails({
+  template,
+  node,
+}: {
+  template: JobTemplate;
+  node: GraphNodeData;
+}) {
+  const { launch_data: promptValues, resource: nodeValues } = node;
+  const { data: nodeLabels } = useGet<AwxItemsResponse<ILabel>>(nodeValues?.related?.labels);
```

Comment:
There are several requests being made here, do we need to add loading state for them?

Reply:
Resolving. We chatted about this and decided it wasn't ideal to show a spinner for a subsection of the details. We could make the entire panel show a spinner but I would prefer to handle that in a separate issue.